### DISCUSSION TITLE
Is there's a way to use hybrid database using prisma?

### DISCUSSION THREAD

**Discussion by 0wx** [prisma/prisma#14391] [Q&A] [answered]
I have been using prisma for a while in my private project, I want to recode my other project code base, the thing is I use hybrid database, mongodb and postgre, I use mongoose and typeorm for these 2, just curious can prisma single handedly do both of these?

**Comment by nurul3101** [comment, marked as answer]
Hey @0wx 👋 
You could use multiple databases with prisma, each of your database would have a separate Prisma Schema and a separate `PrismaClient` Instance.
Please have a look at this proposed [solution](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
